Library/Homebrew/utils/curl.rb

```diff
@@ -42,13 +42,15 @@ def curl_args(*extra_args, show_output: false, user_agent: :default)
   args + extra_args
 end
 
-def curl(*args)
+def curl(*args, **options)
```

Comment:
Could this be a keyword argument instead?

Reply:
Updated